### PR TITLE
Add Kickoff Only script

### PR DESCRIPTION
**Pull request by VirxEC**
[Demo video](https://www.youtube.com/watch?v=hJccrb6abj8)

### CODE REVIEW

File: RLBotPack/KickoffOnly/kickoff_only.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                    continue
+
+                ball_y_side = -1 if packet.game_ball.physics.location.y < 0 else 1
+                ball_state = BallState(Physics(location=Vector3(0, 5500 * ball_y_side, 325), velocity=Vector3(0, 0, 0), angular_velocity=Vector3(0, 0, 0)))
+                self.set_game_state(GameState(ball=ball_state))
+                self.handled = True
+
+
+if __name__ == "__main__":
+    zero_g_mutator = KickoffOnly()
```
Comment (oxrock) [inline RLBotPack/KickoffOnly/kickoff_only.py:41]: although technically not wrong, why is it named zero_g_mutator? I'm all for self documenting variable names but this is misleading.

File: RLBotPack/KickoffOnly/kickoff_only.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import time
```
Comment (oxrock) [inline RLBotPack/KickoffOnly/kickoff_only.py:1]: is this import actually used anywhere? I'm not seeing it.

**Review by NicEastvillage — APPROVED** [review]
Beside what oxrock mentioned, the phrasing in the [Details] section is a bit weird, but otherwise looks good.

File: RLBotPack/KickoffOnly/kickoff_only.py (hunk 1 above)
Comment (VirxEC) [reply RLBotPack/KickoffOnly/kickoff_only.py:41]: lmao I just got exposed

**Pull request merged**